dataplane/pkg/tests/memif_proxy_test.go

```diff
@@ -0,0 +1,18 @@
+package tests
+
+import (
+	"github.com/networkservicemesh/networkservicemesh/dataplane/pkg/memifproxy"
+	. "github.com/onsi/gomega"
+	"testing"
+)
+
+func TestMemifProxy(t *testing.T) {
```

Comment:
Is it possible to add more robust test? 
It will start 2 goroutines: one for writing to `source.sock` and another for reading from `target.sock`. It would be great to have such test :)

Reply:
This test shows that the proxy correctly closing the socket and can reuse it again.

Reply:
But you are right. I'll add another test to testing communication between sockets.

Reply:
Added test and a fixed problem that transfer does not closing on proxy stopping. Thanks!